src/javascripts/legacy_app.js

```diff
@@ -1,8 +1,8 @@
 import BaseApp from 'base_app';
 
-// remove these if not needed
```

Comment:
I'd rather disable the linter around this rule for the branch we are using for migrating apps. For from-scratch apps it's reasonable to comment it.